spp_event_demo/models/house_visit.py

```diff
@@ -27,3 +38,187 @@ def get_view_id(self):
             .search([("model", "=", self._name), ("type", "=", "form")], limit=1)
             .id
         )
+
+
+class OpenSPPHouseVisitRaw(models.Model):
+    _name = "spp.event.house.visit.raw"
+    _description = "House Visit Raw Data"
+    _inherit = ["spp.event.house.visit.mixin"]
+
+    registrant_given_name = fields.Char()
+    registrant_family_name = fields.Char()
+    registrant_id = fields.Many2one("res.partner")
+    state = fields.Selection(
+        [
+            ("New", "New"),
+            ("Validated", "Validated"),
+            ("Error", "Error"),
+            ("Updated", "Updated"),
+            ("Posted", "Posted"),
+        ],
+        "Status",
+        readonly=True,
+        default="New",
+    )
+    remarks = fields.Char()
+    import_id = fields.Many2one("spp.event.data.import")
+
+
+class OpenSPPEventDataImport(models.Model):
+    _inherit = "spp.event.data.import"
+
+    spp_event_house_visit = fields.One2many("spp.event.house.visit.raw", "import_id")
+
+    def spp_event_house_visit_import(self, sheet):  # noqa: C901
+        for rec in self:
+            vals = []
+            columns = []
+            mainvals = {}
+            for row in range(sheet.nrows):
```

Comment:
You can use range(1, sheet.nrows) so that the iteration will start at 1 instead of 0